Moonlight macOS/AppDelegate.m

```diff
@@ -0,0 +1,212 @@
+//
```

Comment:
Can we share some or all of this code with iOS?

Reply:
Yes, done.